pkg/cmd/pulumi/util.go

```diff
@@ -190,14 +190,15 @@ func createSecretsManager(
 	}
 
 	if secretsProvider == passphrase.Type {
-		if _, pharseErr := filestate.NewPassphraseSecretsManager(stack.Ref().Name(), configFile,
-			rotatePassphraseSecretsProvider); pharseErr != nil {
+		if _, pharseErr := filestate.NewPassphraseSecretsManager(stack.Ref().Name(),
```

Comment:
Did you mean `phraseErr` here?

Reply:
Hah I mean I didn't change that text, but yes I'm sure that's what the original author intended. I'll fix it up.